node/src/components/rpc_server/rpcs/info.rs

```diff
@@ -102,6 +107,12 @@ pub struct GetDeployResult {
     pub deploy: Deploy,
     /// The map of block hash to execution result.
     pub execution_results: Vec<JsonExecutionResult>,
+    /// The hash of this deploy's block.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub block_hash: Option<BlockHash>,
+    /// The height of this deploy's block.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub block_height: Option<u64>,
```

Comment:
Done. I the descriptions were missing so I added them with `#[schemars(description = "...")]`.

Reply:
They are, but I didn't want to comment `The hash of this deploy's block.` on `BlockHashAndHeight` because the struct itself, `BlockHashAndHeight`, is not really related to deploys in any way.